lib/SelectableItem/SelectableItem.less

```diff
@@ -3,16 +3,13 @@
 	padding-left: (@moon-item-indent + @moon-spotlight-outset);
 
 	&:before {
-		content: '';
+		content: @moon-icon-circle;
 		position: absolute;
+		top: 0;
```

Comment:
Do we need to specify `top` here? The rest of the changes look great. Thanks for finding and using the variables too. (You can just commit and push to this branch, and the PR will update. No need to close and reopen. 😃)